clusterloader2/pkg/util/util.go

```diff
@@ -129,11 +134,33 @@ func getMap(dict map[string]interface{}, key string) (map[string]interface{}, er
 
 	mapValue, ok := value.(map[string]interface{})
 	if !ok {
-		return nil, fmt.Errorf("type assertion error: %v is not a string", value)
```

Comment:
Why this change?

Reply:
Ach, sorry, you are right. I've confused this part of a code with different one that was recently touched.

You are correct, we are casting to map[string]interface{}.